packages/editor/src/store/reducer.js

```diff
@@ -235,6 +235,60 @@ export const editor = flow( [
 		resetTypes: [ 'SETUP_EDITOR_STATE' ],
 		ignoreTypes: [ 'RECEIVE_BLOCKS', 'RESET_POST', 'UPDATE_POST' ],
 		shouldOverwriteState,
+		serialize: ( present, past ) => {
+			// Return the same reference unless change is necessary.
+			let nextState = present;
+
+			if ( past && past.blocks && present.blocks.byClientId === past.blocks.byClientId ) {
+				return nextState;
+			}
+
+			// For each changed block in the present, check for any transient
+			// attribute. If such an attribute is found, delete it from the
```

Comment:
What's the issue with keeping it in state? Could it be enough to fall under the concept of `withHistory`'s "ignore" behavior?

Related prior art: caed8bf8f9c949c016a4ed14db85de7775c35f96 (#9403)

Reply:
As DM'd, this is problematic due to how we now expect to instantiate blocks with `createBlocks( { myNormalAttr, myTransientAttr } )`.